### PR TITLE
completed shop page and started single page design

### PR DESCRIPTION
**Pull request by rahmanshifur**

### CODE REVIEW

**Comment by rahmanshifur** [comment]
create shop page include in shop-bar, filter-bar, product-list, pagination  and also started single-product page design.

File: components/layout/index.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Header from "./header";
 
 const Layout = (props) => {
     return (
-        <div className="container">
+        <div className="">
```
Comment (naimuddin374) [inline components/layout/index.jsx]: Remove empty class

File: components/product-content/index.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export default ProductContent
+
+
```
Comment (naimuddin374) [inline components/product-content/index.jsx]: remove commented code

File: pages/shop/product-list.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export default ProductList
+
+
+
```
Comment (naimuddin374) [inline pages/shop/product-list.jsx]: remove commented code

**Review by naimuddin374 — CHANGES_REQUESTED** [review]
Change these things

**Review by SohagHossain9983 — COMMENTED** [review]
good job. thank you very much to cganges all reviues

**Review by SohagHossain9983 — COMMENTED** [review]
change empty className

**Review by naimuddin374 — APPROVED** [review]

**Pull request merged**